redfish-core/include/privileges.hpp

```diff
@@ -155,7 +155,7 @@ class Privileges
         size_t endIndex = basePrivilegeCount;
         if (type == PrivilegeType::OEM)
         {
-            searchIndex = basePrivilegeCount - 1;
+            searchIndex = basePrivilegeCount;
```

Comment:
Why change this?

Reply:
Oops.  This is to fix a code bug in handling OEM privileges (which BMCWeb does not yet have upstream).  Note that basePrivilegeCount is 5, and the search for OEM privileges should start at index 5, not 4.  Off by one.
I created https://gerrit.openbmc-project.xyz/c/openbmc/bmcweb/+/46154 to merge this upstream.

Reply:
Cool. Thanks for doing that. I merged https://gerrit.openbmc-project.xyz/c/openbmc/bmcweb/+/46154